### ISSUE TITLE
Preconfigure connection with json file

### ISSUE THREAD

**Issue by Mrxx99** [nats-nui/nui#27]
It would be nice if it would be possible to be able to configure connections via a environment variable or docker run args or a config file that can be mapped into the container.
So it doesn't have to be configured manually in the UI.
Would be useful especially when using something like docker compose.

**Comment by pricelessrabbit** [comment]
The original idea was to directly support nats cli context and load from that json, but theo other features tooks the priority. Configuring every aspect of the connection (auth included) via env is messy, so i will take into consideration to load from config file or nats context. 
For now, if you have a lot of connections or you want to automate it, maybe you can directly use a script and the rest api to load all the connections when you start the service for the first time

**Comment by Mrxx99** [comment]
agree, configuring via env could get messy unless only simply connection strings with token or username/passwords are used. But my second suggestion by mapping in a json file in a docker volume/mount that contains the information would not be messy and be useful for docker compose kubernetes deployments.
It would be much easier than using a script and rest api that I would not know how to use a single json file definition would be easier to unserstand by just giving on example file.

**Comment by Mrxx99** [comment]
I could achieve my use case using the /api/connection API, thanks for that tip. Is that API stable? If so, could some small documentation be added for it? Mapping a file could still be useful but not required for my use case. If you want you can close the issue.

**Comment by pricelessrabbit** [comment]
About the API, the connection one is pretty stable (but keep in mind is not meant to be a "public" api so additions / updates can happen.) you can check the api doc [here ](https://github.com/nats-nui/nui/tree/main/frontend/docs/entities). In particular if your goal is to define an initial set of connections, you should look [here](https://github.com/nats-nui/nui/blob/main/frontend/docs/entities/connection/def/connection.md) for the entity and [here](https://github.com/nats-nui/nui/blob/main/frontend/docs/entities/connection/create.md) for the path. All of the payloads are documented in a typescript-like syntax but the api is obvously in json.
I'm keeping the issue open because the idea of some kind of config import is good,  i only need to find out some time to implement it
